src/modules/claim/claim.service.ts

```diff
@@ -140,6 +140,13 @@ export class ClaimService {
     const updatedClaim = Claim.create({ ...claim, ...data, isAccepted: true });
     return this.claimRepository.save(updatedClaim);
   }
+  
+  /**
+   * returns all claims
```

Comment:
@JGiter Could you clarify what you mean by `all claims` here? Is this "all claims for a specific DID" or "maybe all claims for the currently signed-in DID"?

Reply:
It returns all claims of all DID's. I've updated comment and PR description